dags/hindawi/parser.py

```diff
@@ -0,0 +1,229 @@
+import re
+import xml.etree.ElementTree as ET
+
+from common.parsing.parser import IParser
+from common.parsing.xml_extractors import CustomExtractor, TextExtractor
+from structlog import get_logger
+from common.parsing.extractor import IExtractor
+
+
+class HindawiParser(IParser):
+    prefixes = {
+            "marc": "http://www.loc.gov/MARC21/slim",
+            "ns0": "http://www.openarchives.org/OAI/2.0/",
+            "ns1": "http://www.loc.gov/MARC21/slim",
+            "xsi": "http://www.w3.org/2001/XMLSchema-instance",
+        }
```

Comment:
```suggestion
    prefixes = {
            "marc": "http://www.loc.gov/MARC21/slim",
            "ns0": "http://www.openarchives.org/OAI/2.0/",
            "ns1": "http://www.loc.gov/MARC21/slim",
            "xsi": "http://www.w3.org/2001/XMLSchema-instance",
     }
```


This should be fixed with pre-commit, have you install it? 😅